librz/include/rz_magic.h

```diff
@@ -5,6 +5,13 @@
 #define RZ_MAGIC_H
 
 #include <rz_types.h>
+#include <sys/types.h>
+#include <rz_util/rz_rbtree.h>
+#include <sys/queue.h>
+
+#include <regex.h>
```

Comment:
Is this `regex.h` header present on all systems?
It looks like the regex matching is done only one time in `magic-load.c::260`.
Maybe we can use our library?

Reply:
You would use `rz_regex_new`, `rz_regex_free`, and `rz_regex_match` for the matching.